mods/d2k/rules/arrakis.yaml

```diff
@@ -1,22 +1,25 @@
 spicebloom.spawnpoint:
-	HiddenUnderShroud:
-		Type: CenterPosition
+	EditorOnlyTooltip:
+		Name: Spice Bloom spawnpoint
+	AlwaysVisible:
+	RenderSpritesEditorOnly:
+		Image: spicebloom
```

Comment:
I would love to see that too, it is too hard to see spawnpoints after placed them currently.